network-config-analyzer/ExecutionAssist.py

```diff
@@ -131,7 +131,7 @@ def execute(self):
         print()
         full_result = ContainmentQuery(self.permit_config, self.base_config).exec(True)
         if not full_result.bool_result:
-            print(f'{self.base_config.name} does not permit connections specified in {self.permit_config.name}:')
+            print(f'{self.base_config.name} does not explicitly permit connections specified in {self.permit_config.name}:')
```

Comment:
```suggestion
            print(f'{self.base_config.name} does not permit connections specified in {self.permit_config.name}:')
```
It's also ok if `self.base_config` permits the connections implicitly.

Reply:
done